src/auth/dto/create-qrcode-2fa-response.ts

```diff
@@ -0,0 +1,12 @@
+export class CreateQrcodeFor2FA {
+  public qrcodeUrl: string;
+  public urlToEnable2FA = 'http://localhost:3000/api/auth/enable-2fa';
```

Comment:
`http://localhost:3000` part should be read from environment variable. The current one will not work on staging.